### PR TITLE
util for searching standards against procedure docs

### PR DESCRIPTION
**Pull request by joedimarzio**

### CODE REVIEW

File: util/compliance.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+from os import path, rename, mkdir, listdir
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize
+from nltk import download
+from whoosh.index import create_in, open_dir
+from whoosh.fields import Schema, TEXT
+from whoosh.qparser import QueryParser
+from whoosh import scoring
+
+directory = '/Users/jpdimarz/Repos/security-policy-templates/templates/procedures' #set this to the appropriate path
```
Comment (aiwilliams) [inline util/compliance.py]: Could this be made an argument, defaulting to something relative to the project? For example:

```sh
$ git clone git@github.com:JupiterOne/security-policy-templates.git
$ cd security-policy-templates
$ python3 util/compliance.py
```

Otherwise, we'd have to edit this file to run this file, and then revert the change to the working directly clean for any commits.

File: util/compliance.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+standards_file = 'csa-ccm.json' #set this to the appropriate file
+#download('punkt')  these must be downloaded the first time the script is ran
+#download('stopwords')
```
Comment (aiwilliams) [inline util/compliance.py]: Perhaps something like [`asdf`](https://github.com/danhper/asdf-python) could be encouraged for Python version management, and then `pip` for dependencies, or whatever the latest best practices are? It would be nice to avoid having to read this source code to find these comments and get something installed by uncommenting, then re-commenting etc.

File: util/compliance.py (hunk 1 above)
Comment (joedimarzio) [reply util/compliance.py]: Good point. I changed it to use argparse and added a README for documentation.

File: util/compliance.py (hunk 2 above)
Comment (joedimarzio) [reply util/compliance.py]: I added a script that uses pip to install all dependencies. I looked into Python packaging via setuptools and it's somewhat of a process, so I don't think it's necessary considering it's just a util script. If we develop it further or decide that it's useful enough I could package it properly.
